File: Assets/Typescript~/src/Shared/BedWars/BedWars.ts
const scene: string = SceneManager.GetActiveScene().name;

export class BedWars {
	public static IsLobbyMode(): boolean {
		return scene.lower().find("bwlobbyscene")[0] !== undefined;
	}

	public static IsMatchMode(): boolean {
		return scene.lower().find("bwmatchscene")[0] !== undefined;
	}

	public static GameId = "6536ee084c9987573c3a3c03";
}

File: Assets/Bundles/Shared/Resources/TS/BedWars/BedWars.lua
-- Compiled with unity-ts v2.1.0-75
local scene = SceneManager:GetActiveScene().name
local BedWars
do
	BedWars = setmetatable({}, {
		__tostring = function()
			return "BedWars"
		end,
	})
	BedWars.__index = BedWars
	function BedWars.new(...)
		local self = setmetatable({}, BedWars)
		return self:constructor(...) or self
	end
	function BedWars:constructor()
	end
	function BedWars:IsLobbyServer()
		return scene == "BWLobbyScene"
	end
	function BedWars:IsMatchServer()
		return scene == "BWMatchScene"
	end
end
return {
	BedWars = BedWars,
}
-- ----------------------------------
-- ----------------------------------
